Automattic/wp-calypso
# Reader: add controller and route for /read/start

Following on from the split of Reader sections into separate controllers in #5088, this PR adds a controller and route for the upcoming `/read/start`.

## To test

1. Navigate to http://calypso.localhost:3000/read/start in development
2. Verify that 'Welcome to the Reader' is displayed
